erpnext/stock/doctype/material_request_item/material_request_item.json

```diff
@@ -217,11 +217,10 @@
   },
   {
    "fieldname": "item_group",
-   "fieldtype": "Link",
+   "fieldtype": "Data",
```

Comment:
https://github.com/frappe/erpnext/issues/26447 

Referring to your original issue a better solution could be to "disable" an item group so no new items can be added to it. 

I don't think throwing away references or leaving stale references like this is a good idea. 

In MR it _might_ be okay.  In invoices? A big no. It's used by many reports. 

---

aside: please follow contribution guidelines regarding PR title and semantic pr/commits.

Reply:
Thanks for your feedback. As this structure is used all over the system I will take that into account and now know this for future reference.

I'm trying to thing of any reason why I would need any more than a text reference in all the transactions. I see that it makes navigation through the system easier as a link is clickabkle and the data is not. But beyond that? Beeing able to see all the transaction linked to the item group rather than finding that information at a customer/suppliert or item level.

Best